### PR TITLE
MYR-74 : rocksdb.rpl_row_not_found fails

### PR DESCRIPTION
**Pull request by george-lorch**
- When running with full debug build and --debug-server, extra suppressions are
  needed for log file.

### CODE REVIEW

**Comment by george-lorch** [comment]
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/808/ shows the issue fixed

File: mysql-test/suite/rocksdb/include/rpl_row_not_found.inc [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 connection slave;
 call mtr.add_suppression("Slave SQL: Worker.*failed executing transaction");
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/include/rpl_row_not_found.inc]: Is this one still needed? The one added below looks too similar, is it as 5.6 vs 5.7 difference?

Comment (george-lorch) [reply mysql-test/suite/rocksdb/include/rpl_row_not_found.inc]: I think that is all it is, a 5.6 to 5.7 diff with 5.7 bringing in the "for channel" but can't say offhand for sure as I have not gone back and looked at the actual 5.6 error being logged.

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Replace the suppression instead of adding a new one

**Comment by george-lorch** [comment]
Yup, will do, it just didn't occur to me that the current was 5.6 specific.

**Comment by george-lorch** [comment]
OK, re-pushed with the deprecated suppression removed and test re-recorded. jenkins here http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/811/ and we can see the test passing.

I'm going to go ahead and merge it at this point, nothing more really to do.

**Pull request merged**